satpy/etc/composites/abi.yaml

```diff
@@ -147,3 +147,333 @@ composites:
         - name: C13
     - name: C08
     standard_name: airmass
+
+###
+## Airmass using Channel14 RGB (own)
+  airmass_eum:
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - compositor: !!python/name:satpy.composites.DifferenceCompositor
+      prerequisites:
+      - name: C08
+      - name: C10
+    - compositor: !!python/name:satpy.composites.DifferenceCompositor
+      prerequisites:
+        - name: C12
+        - name: C14
+    - name: C08
+    standard_name: airmass
+
+  night_microphysics:
+    compositor: !!python/name:satpy.composites.Dust
+    prerequisites:
+    - name: C07
+    - name: C14
+    - name: C15
+    standard_name: night_microphysics
+
+  convection:
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - compositor: !!python/name:satpy.composites.DifferenceCompositor
+      prerequisites:
+        - name: C08
+        - name: C10
+    - compositor: !!python/name:satpy.composites.DifferenceCompositor
+      prerequisites:
+        - name: C07
+        - name: C14
+    - compositor: !!python/name:satpy.composites.DifferenceCompositor
+      prerequisites:
+        - name: C05
+        - name: C02
+    standard_name: convection
+
+## Cloud Phase Distinction or Day Cloud Phase (name from NASA SPORT) or Cloud Type RGB(name from EUMETSAT)
+  cloud_phase_distinction_raw:
+    compositor: !!python/name:satpy.composites.SelfSharpenedRGB
+    prerequisites:
+    - name: C13
+    - name: C02
```

Comment:
To properly use `SelfSharpenedRGB` I think you need to include `high_resolution_band: green` as a keyword argument (same indentation level as `standard_name` below). Otherwise it will try to do the sharpening based on the "red" band by default which is not the high resolution band.

There is at least this use of `SelfSharpenedRGB` and the composite below. Maybe others.